test/unit/processing/statement/get-statement.test.js

```diff
@@ -72,8 +88,19 @@ describe('get various components and transform to statement object', () => {
 
   test('should call getCalculation when a paymentRequestId is given', async () => {
     const paymentRequestId = 1
-    const statement = await getStatement(paymentRequestId)
-    console.log(statement)
+    await getStatement(paymentRequestId)
     expect(getCalculation).toHaveBeenCalled()
   })
+
+  test('should call getLastSettlement one when a paymentRequestId is given', async () => {
+    const paymentRequestId = 1
+    await getStatement(paymentRequestId)
+    expect(getLastSettlement).toHaveBeenCalledTimes(1)
+  })
+
+  test('should call getLatestPayment when a paymentRequestId is given', async () => {

```

Comment:
should call with

Reply:
I didn't add those because the mock setup of this test returns objects that don't match what the actual functions pass to.

I think we need to scrap this test suite and re-write it.

Unrelated I also think we need an integration test that covers this flow as there currently are no tests that cover the orchestration between these functions that don't use mocks.